rest-api/test/unit_test/api_test/biobank_order_api_test.py

```diff
@@ -17,9 +19,28 @@ def setUp(self):
     self.participant_dao = ParticipantDao()
     self.participant_dao.insert(self.participant)
     self.summary_dao = ParticipantSummaryDao()
+    self.bio_dao = BiobankOrderDao()
     self.path = (
         'Participant/%s/BiobankOrder' % to_client_participant_id(self.participant.participantId))
 
+  def test_cancel_order(self):
+    self.summary_dao.insert(self.participant_summary(self.participant))
+    order_json = load_biobank_order_json(self.participant.participantId,
+                                         filename='biobank_order_2.json')
+    result = self.send_post(self.path, order_json)
+    full_order_json = load_biobank_order_json(self.participant.participantId,
+                                              filename='biobank_order_1.json')
+    _strip_fields(result)
+    _strip_fields(full_order_json)
+    self.assertEquals(full_order_json, result)
+
+
+    # @TODO: in progress.
```

Comment:
We definitely should have tests of the new PUT method before checking this in. You could just merge this in with the next PR I guess, so you can test out the version stuff too.